tools/README.md

```diff
@@ -0,0 +1,94 @@
+# How to Build
+
+## Requirements
+- Update the [`CHANGELOG`](../CHANGELOG.md)
+- Update the `version` in [`package.json`](../package.json#L3)
+- Update the `version` in [`logdna-agent.rb`](./files/darwin/logdna-agent.rb#L2)
```

Comment:
Sorry, I don't understand this.  Can you explain what the updating is for?

Reply:
@darinspivey, basically `logdna-agent.rb` serves as `package.json` for `MacOSX` / `Darwin` systems since in order to update the package we need to create a PR for `Homebrew Cask` containing the updated `logdna-agent.rb`

`logdna-agent.nuspec` does the same for `Windows` / `Win32`